src/selectedtests/work_items/cli.py

```diff
@@ -0,0 +1,66 @@
+"""Cli entry point to process work items."""
+import click
+import os
+import structlog
+import logging
+import pytz
+
+from datetime import datetime, timedelta
+from evergreen.api import RetryingEvergreenApi
+from evergreen.config import EvgAuth
+from selectedtests.work_items.process_work_items import process_queued_work_items
+from selectedtests.datasource.mongo_wrapper import MongoWrapper
+
+
+def _get_evg_api():
```

Comment:
[nit] We might consider pulling some of these helper functions into a file that can be shared with other cli entrypoints. It looks like we could reuse them.